books/the-davinci-code.json

```diff
@@ -0,0 +1,18 @@
+{
+    "title": "The Da Vinci Code",
+    "storyline": "Symbologist Robert Langdon travels from Paris to London to discover the truth behind a mysterious and bizarre murder. Later, he learns about a religious mystery protected by a secret society.",
+    "authors": ["Dan Brown"],
+    "genres": [
+      "quest",
+      "adventure",
+      "mystery",
+      "thriller",
+      "history"
+    ],
+    "characters": ["Robert Langdon","Sophie Neveu","Silus","Sir Leigh Teabing"],
+    "pages": 489,
+    "ISBN": "9781435288232",
+    "language": "english",
+    "publisher": "Doubleday (US) Transworld & Bantam Books (UK)",
+    "publication_date": "April 2003"
```

Comment:
`publication_date` is incorrect. It should be in the following format:
https://github.com/404minds/books-db/blob/a49ed698b7a84e69f32d5e6a9c99c41b6f96cd25/books/1984.json#L20

Reply:
Noted. Kindly review the changes made, thanks!